selftests/functional/test_output.py

```diff
@@ -475,6 +475,10 @@ def test_results_plugins_no_tests(self):
         tap_results = os.path.join(self.tmpdir, 'latest', 'results.tap')
         self.assertFalse(os.path.exists(tap_results))
 
+        # Check that no UI output was generated
+        self.assertNotIn("RESULTS    : PASS ", result.stdout)
+        self.assertNotIn("JOB TIME   :", result.stdout)
```

Comment:
Yeah, I can add that to the other commit (where the "Error running" message is actually generated).